src/types.ts

```diff
@@ -1,3 +1,5 @@
+import { CSSProperties } from 'react'
```

Comment:
This import isn't necessary — it currently throws a warning when running `npm run build`